linked_list/linked_list.cpp

```diff
@@ -0,0 +1,325 @@
+/*
+* C++ Program to Implement Linked List
+*/
+#include <iostream>
+#include <cstdio>
+#include <cstdlib>
+
+using namespace std;
+/*
+* Node Declaration
+*/
+struct node
+{
+	int info;
+	struct node *next;
+}*start;
+
```

Comment:
>structs should be used for data only
For struct "single_list"?
Then what is the benefit of using C++?
Take a look at the java implementation of Linked List.
https://github.com/iiitv/algos/blob/master/linked_list/LinkedList.java